game.py

```diff
@@ -20,7 +20,31 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    wrong_guesses_count = 0
+    wrong_guesses_list = []
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    
+    while wrong_guesses_count < SNOWMAN_MAX_WRONG_GUESSES:
+        user_input = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
```

Comment:
This line of characters exceeds (due to the indentation) the suggested 80 character limit for readability. You'd want to use the [PEP 8 Style Guidelines for line continuation](https://peps.python.org/pep-0008/#indentation) to break this up onto multiple lines. 
```suggestion
        user_input = get_letter_from_user(
            correct_letter_guess_statuses,
            wrong_guesses_list)
```